aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js

```diff
@@ -23,21 +23,19 @@ export const NameValueListEditExample = () => {
 
   return (
     <Box gap="medium">
-      <Box justify="between" width="large" direction="row">
-        <Heading level={2} size="small" margin="none">
-          Profile Details
-        </Heading>
-        {!edit && (
-          <Box justify="center">
+      <PageHeader
+        title="Profile Details"
+        actions={
```

Comment:
Does the included Button need the `alignSelf="center"`? Should it be removed?

Reply:
I think it can be removed, thanks for noting.

Reply:
Done.